linkedlist/singly/python/SinglyLinkedList.py

```diff
@@ -70,20 +70,19 @@ def addLast(self, element):
         :param element: the value held in the Node.
         """
 
-        # Starting at the head node, travserse the linkedlist until we get to the tail node
```

Comment:
Hey, pls undo this change. I really like that you thought about this, but you'd need to add an explanation for this in the readme first and a time complexity explanation. All the implementations follow the pseudocode laid out in the READMEs.

another suggestion is to create a different method that implements this in constant time. Maybe call it `addLastConstantTime` for brevity. So that whoever reads through will. see that `addLast` can be done in a faster time.  @OsasAzamegbe 

FYI @Youngestdev

Reply:
I have moved the constant time implementation to a separate method "addLastConstantTime()". I also added the pseudocode to the readme file.

Reply:
Ah, that's true. I think a different method is more suitable 👍🏾

Reply:
I created a different PR with just the addLastConstantTime() method and the readme update, to separate it from the queue implementation. Feel free to close this PR at any time.